onnx/defs/rnn/defs.cc

```diff
@@ -105,20 +123,34 @@ implementation such as CuDNN.
 `RB[zrh]` - R recurrence weight matrix for backward update, reset, and hidden gates
 `WBb[zrh]` - W bias vectors for backward update, reset, and hidden gates
 `RBb[zrh]` - R bias vectors for backward update, reset, and hidden gates
-`tanh(X)` - hyperbolic tangent of X
-`sigmoid(X)` - 1 / (1 + e^-X)
 `H` - Hidden state
 `num_directions` - 2 if direction == bidirectional else 1
 
+Activation functions:
+  ReLU(x)                - max(0, x)
+  tanh(x)                - (1 - e^{-2x})/(1 + e^{-2x})
+  sigmoid(x)             - 1/(1 + e^{-x})
+  (Below are optional)
+  linear(x)              - alpha*x + beta
+  leakyReLU(x)           - x if x >= 0 else alpha * x
+  thresholdedReLU(x)     - x if x >= alpha else 0
+  PReLU(xi)              - xi if xi >= 0 else alpha[i]* xi over dim 0
+  scaledTanh(x)          - alpha*tanh(beta*x)
+  sigmoidHard(x)         - min(max(alpha*x + beta, 0), 1)
+  ELU(x)                 - x if x >= 0 else alpha*(e^x - 1)
+  softsign(x)            - x/(1 + |x|)
+  softplus(x)            - log(1 + e^x)
+  parametricSoftplus(xi) - alpha[i]*log(1 + e^{beta[i]* xi}) over dim 0
+
 Equations (GRU with default activations):
-  - zt = sigmoid(Wz*Xt + Rz*Ht-1 + Wbz + Rbz)
-  - rt = sigmoid(Wr*Xt + Rr*Ht-1 + Wbr + Rbr)
-  - ht = tanh(Wh*Xt + rt*(Rh*Ht-1 + Rbh) + Wbh)
-  - H = (1 - zt) (.) ht + it (.) Ht-1
+  - zt = sigmoid((Xt^T)*Wz + Ht-1*Rz + Wbz + Rbz)
```

Comment:
I'd replace sigmoid with f(), and tanh with g(). This way we understand better why 2 / 4 functions are needed.

Reply:
Done.